import-service/src/api/import-products-file.ts

```diff
@@ -0,0 +1,48 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import 'source-map-support/register';
+import { getLog } from './helpers/get-log';
+import { S3 } from 'aws-sdk';
+import { jsonResult } from './helpers/json-result';
+import { validate } from 'jsonschema';
+import { invalidInputError } from './errors/invalid-input.error';
+import { cannotImportFileError } from './errors/cannot-import-file.error';
+
+export const importProductsFile: APIGatewayProxyHandler = async (
```

Comment:
**thought**: in this function never used `await`, maybe it was missed immediately before invocation `s3.getSignedUrl`?

Reply:
Baulderdash. `getSignedUrl` returns synchronously.